.gitignore

```diff
@@ -1,4 +1,222 @@
-build/
-node_modules/
+### Typescript ###
+dist/
```

Comment:
Really? Sure, no problem. But I really wouldn't rely on users' global settings. I'd rather have some extra harmless lines on a text file that making a contributor change his PR because he involuntary pushed some `.Trash-343dsf` or `~RxChannel.ts` file.